### PR TITLE
update P2P.ORG Validator in Mainnet

### PR DESCRIPTION
**Pull request by raven428**

### CODE REVIEW

File: networks.json [hunk 1]
```diff
@@ -2,7 +2,6 @@
   {
     "name": "mainnet1",
     "configFileUrl": "https://raw.githubusercontent.com/vegaprotocol/networks/master/mainnet1/mainnet1.toml",
-    "explorer": "https://explorer.vega.xyz/",
```
Comment (jeremyletang) [inline networks.json:5]: Wht remove the link to the explorer here?

**Review by jeremyletang — CHANGES_REQUESTED** [review]
Could you add back the link to the explorer in the networks.json file? Happy to merge this PR once this is done.

Thanks!

File: networks.json (hunk 1 above)
Comment (raven428) [reply networks.json:5]: IDK why this commit has appeared. It's completely not mine, as you can see there is no signature on this commit

**Comment by raven428** [comment]
Created new pull request due this closed for some reason

**Pull request closed without merging**